README.md

```diff
@@ -410,37 +466,131 @@ function onThingHappended(cb){
 
     ```javascript
     // bad
-    var superman = {
+    const superman = {
       class: 'alien'
     }
 
     // bad
-    var superman = {
+    const superman = {
       klass: 'alien'
     }
 
     // good
-    var superman = {
+    const superman = {
       type: 'alien'
     }
     ```
 
+- Use computed property names when creating objects with dynamic property names.
+
+  > Why? They allow you to define all the properties of an object in one place.
+
+    ```javascript
+
+    function getKey(k) {
+      return `a key named ${k}`
+    }
+
+    // bad
+    const obj = {
+      id: 5,
+      name: 'San Francisco'
+    }
+    obj[getKey('enabled')] = true
+
+    // good
+    const obj = {
+      id: 5,
+      name: 'San Francisco',
+      [getKey('enabled')]: true
+    }
+    ```
+
+  - Use object method shorthand.
+
+    ```javascript
+    // bad
+    const atom = {
+      value: 1,
+
+      addValue: function addValue (value) {
+        return atom.value + value
+      }
+    }
+
+    // good
+    const atom = {
+      value: 1,
+
+      addValue(value) {
+        return atom.value + value
+      }
+    }
+    ```
+
+  - Use property value shorthand.
+
+  > Why? It is shorter to write and descriptive.
+
+    ```javascript
+    const lukeSkywalker = 'Luke Skywalker'
+
+    // bad
+    const obj = {
+      lukeSkywalker: lukeSkywalker
+    }
+
+    // good
+    const obj = {
+      lukeSkywalker
```

Comment:
yea, it's pretty weird at first, but then you start using it, and it's really nice. 

```js
// this sucks
return {firstName: firstName, lastName: lastName, age: age}
// this is much nicer
return {firstName, lastName, age}
```